tests/atest/transformers/OrderTags/expected/case_insensitive_reverse.robot

```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+Documentation       OrderTags acceptance tests
+Force Tags          forced_tag_Bb    forced_tag_ba    forced_tag_Ab    forced_tag_aa    forced_tag_2    forced_tag_1
```

Comment:
More ideas for testing - comments:
```
Force Tags    forced_tag_Bd   # i should be preserved
```
Multiline (it will be converted to one line but it's fine):
```
[Tags]    i
...    span
...    multiple
....   lines
```

Reply:
Added multiline tests to both OderTags and NormalizeTags transformers.
I am not sure how to preserve comments, could you take care of it or suggest some solution?